scripts/deploy-workbook.sh

```diff
@@ -0,0 +1,213 @@
+#!/bin/sh
```

Comment:
I would like a customer to use Cloud Shell, WSL or pure Linux for now. And, to clone on Windows environment is not assumed because it would lead problem like return code.
In next release or future release, I'd like to develop another tool to run any environment, e.g. Golang with Azure SDK to solve this problem.
ShellScript is easy to develop and understand while targeting small case and environment, but it's difficult to handle for supporting many environment.

How do you feel that in this release, I'd like to target specific environment?

Reply:
Yes, that should be fine. We can callout to run on Bash option in Cloudshell.